definitions/npm/react-jss_v6.1.x/test_react-jss_v6.1.x.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+import React from 'react';
+import injectSheet, { create } from 'react-jss';
+import type { Classes } from 'react-jss';
+
+
+const styles = {
+  root: {
+    backgroundColor: 'red',
+  },
+  [`@media (min-width: ${10}px)`]: {
+    root: {
+      width: 200
+    }
+  }
+};
+
+type Props = {
+  classes: Classes<typeof styles>,
+  content: string,
+}
+
+const FunctionComponent = (props: Props) => (
+  <div className={props.classes.root}>{props.content}</div>
+);
+
+const FunctionComponentUsesWrongClassname = (props: Props) => {
+  // $ExpectError - property `nonExistentClassName` is not found in "styles"
+  return <div className={props.classes.nonExistentClassName}>{props.content}</div>;
+};
+
+class ClassComponent extends React.Component {
+  props: Props;
+
+  render() {
+    const { classes, content } = this.props;
+
+    return <div className={classes.root}>{content}</div>
+  }
+}
+
+
+// ===================================
+// "create" signature
+// ===================================
+
+const customInjectSheet = create();
+
+// $ExpectError - missing "styles" argument
+customInjectSheet()(FunctionComponent);
+
+// $ExpectError - wrong type of "styles" argument
+customInjectSheet(123)(FunctionComponent);
+
+// no errors
+customInjectSheet(styles)(FunctionComponent);
+
+
+// ===================================
+// "injectSheet" signature
+// ===================================
+
+// $ExpectError - missing "styles" argument
+injectSheet()(FunctionComponent);
+
+// $ExpectError - wrong type of "styles" argument
+injectSheet(123)(FunctionComponent);
+
+// no errors
+injectSheet(styles)(FunctionComponent);
```

Comment:
a no-render component `render() {return null}` but the sheet is still injected, for e.g. useful if you need global styles only without any dom nodes in that particular component.